maxim/tests/test_logger_langchain_02x.py

```
@@ -1,865 +1,865 @@
+#     def setUp(self):
+#         config = Config(apiKey=apiKey, baseUrl=baseUrl, debug=True)
+#         self.maxim = Maxim(config)
+
+#     def test_generation_chat_prompt(self):
+#         logger = self.maxim.logger(LoggerConfig(id=repoId))
+#         model = OpenAI(
+#             callbacks=[MaximLangchainTracer(logger)], api_key=openAIKey)
+#         messages = [
+#             (
+#                 "system",
+#                 "You are a helpful assistant that translates English to French. Translate the user sentence.",
```

Pylint (3.3.7)

[convention] Line too long (112/100)

(C0301)